client/landing/subscriptions/components/site-list/site-row.tsx

```diff
@@ -68,25 +70,37 @@ export default function SiteRow( {
 		SubscriptionManager.useSiteEmailMeNewCommentsMutation();
 	const { mutate: unsubscribe, isLoading: unsubscribing } =
 		SubscriptionManager.useSiteUnsubscribeMutation();
+
+	const navigate = useNavigate();
+	const locale = useLocale();
+
+	const individualPagePath = useMemo( () => {
+		return `/subscriptions/site/${ blog_ID }` + ( locale !== 'en' ? '/' + locale : '' );
+	}, [ blog_ID, locale ] );
```

Comment:
Is it necessary to include the locale in the URL? 🤔 In my tests it is not necessary; to be more precise, if I keep the locale in the URL, the linking to the individual subscription page doesn't work.

```suggestion
	const individualPagePath = useMemo( () => {
		return `/subscriptions/site/${ blog_ID }`;
	}, [ blog_ID ] );
```

Reply:
Good catch, we'll need to support locales eventually.